### PR TITLE
Ft/get verify passcode

### PR DESCRIPTION
**Pull request by mudulo**

### CODE REVIEW

**Comment by Harry3167** [comment]
@mudulo is the getUserVerificationActivityIntent API on a different PR?

File: TODO.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ This phase will include all actions and activities that enable you to conduct a
 ⬜ Action: Authenticate Passcode Registered User on Program using Form Factor-Card
```
Comment (Harry3167) [inline TODO.md:66]: Please add a green check mark here to show the whole action is completed

**Comment by mudulo** [comment]
@Harry3167 yes

File: docs/api-references/user-lifecycle-management-api.md [hunk 2]
```diff
@@ -255,4 +255,62 @@ In addition to the [general error codes](https://developer.mastercard.com/cp-ker
...
+## 3 User Authentication
+
+### 3.1 getVerifyPasscode
+
+This API is used to verify that the Passcode provided by the user is the same as the one present on the user’s card/CP Consumer Device. in this flow, the Reliant Application takes an input from the user, which consists of 6 digits, and passes it to CPK for passcode verification.
+
+**Compatibility**
+| **Available as of CPK version #** | **Deprecated as of CPK version #** |
+|-----------------------------------|------------------------------------|
+| + CPK 2.0.1 | + n/a |
```
Comment (Harry3167) [inline docs/api-references/user-lifecycle-management-api.md:267]: The wrappers APIs such as getVerifyPasscode are available from CPK 2.4.1

File: src/index.ts [hunk 3]
```diff
@@ -90,6 +90,28 @@ export function getRegistrationData({
...
+
+export interface GetVerifyPasscodeParamType {
+  passcode: string,
+  programGUID: string,
+  reliantGUID: string
+}
+
+export interface GetVerifyPasscodeResultType {
+  status: boolean,
+  rId: string,
```
Comment (Harry3167) [inline src/index.ts:111]: The properties in the interfaces should be separated using semicolons. Please replace the commas with semicolons.

**Review by Harry3167 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 3 above)
Comment (mudulo) [reply src/index.ts]: Noted. Fixed [Here](https://github.com/Mastercard/community-pass-react-native-wrapper/pull/13/commits/bb7f6408f50ee86a8a3f882ff9f4f0bcb4b2bb51)

File: docs/api-references/user-lifecycle-management-api.md (hunk 2 above)
Comment (mudulo) [reply docs/api-references/user-lifecycle-management-api.md]: Noted. Fixed [Here](https://github.com/Mastercard/community-pass-react-native-wrapper/pull/13/commits/bb7f6408f50ee86a8a3f882ff9f4f0bcb4b2bb51)

File: TODO.md (hunk 1 above)
Comment (mudulo) [reply TODO.md]: Noted. Fixed [Here](https://github.com/Mastercard/community-pass-react-native-wrapper/pull/13/commits/bb7f6408f50ee86a8a3f882ff9f4f0bcb4b2bb51)

**Review by Harry3167 — APPROVED** [review]

**Pull request merged**